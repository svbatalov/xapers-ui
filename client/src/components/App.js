import React from 'react';
import { Switch, Route } from 'react-router-dom';
import SearchPanel from './SearchPanel';
import EditPanel from './EditPanel';

export default class App extends React.Component {
  render() {
    return (
      <Switch>
        <Route exact path="/" component={SearchPanel} />
        <Route path="/:id" component={EditPanel} />
      </Switch>
    );
  }
}
