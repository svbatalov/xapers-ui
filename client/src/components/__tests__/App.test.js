import React from 'react';
import { shallow } from 'enzyme';

import App from '../App';

describe('<App />', () => {
  test('render', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
