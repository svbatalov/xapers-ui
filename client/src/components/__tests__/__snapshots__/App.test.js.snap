// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> render 1`] = `
<h1>
  Perfectly simple Webpack 2 setup
</h1>
`;
